src/curriculum/dataflow/dataflow.json

```diff
@@ -1,8 +1,9 @@
 {
+  "code": "dataflow",
   "title": "Dataflow",
   "subtitle": "Computational Thinking in Biology",
   "pageTitle": "%unitTitle%",
-  "code": "dataflow",
+  "demoProblemTitle": "%investigationTitle%",
```

Comment:
Am I correct in assuming that this is because we don't plan on having an Investigation that contains a set of Problems in Dataflow (as per the DF team's request)?  Hence all we'll have is a single Investigation whose title ("Dataflow" at present) will be used on the Demo Creator page?

Reply:
Yes, if you look at `dataflow.json`, there's only one of everything and the title is simply `Dataflow`. The previous code resulted in problems labeled `Dataflow: Dataflow` which didn't make much sense.

Reply:
Right, that's what I saw when testing these changes vs. master.  Makes sense since I don't believe we will need multiple Dataflow problems or Investigations (at least not according to the DF team).  I guess we could just put `"demoProblemTitle": "Dataflow"`, but we might lose some future flexibility.

Reply:
True, but as long as I'm setting up a parameterizable system I might as well make use of it. 😉